app/models/immunisation_import_row.rb

```diff
@@ -140,6 +146,16 @@ def delivery_method
     end
   end
 
+  def dose_sequence
+    return 1 if vaccine.maximum_dose_sequence == 1
```

Comment:
A part of me is feeling paranoid over this bit. It means that there's a codepath where we could overwrite what is in the import file, and I don't think we should ever do that. Sure, theoretically we shouldn't ever hit this since we should only get the `DOSE_SEQUENCE` field when `maximum_dose_sequence` is > 1, but why even allow for the possibility? Is there a benefit to doing the check like this?

I can think of two alternatives that would make me less paranoid:
1. `return 1 if @data.key?("DOSE_SEQUENCE") && vaccine.maximum_dose_sequence == 1` ... a bit lengthy, so there's also the option ...
2. put this into the `rescue`

@benilovj am I being too paranoid?

Reply:
Yeah I think I agree with you, if the file contains `DOSE_SEQUENCE` values we should try and import them even if they can only be one value (`1`).